app/src/entry-point.lisp

```diff
@@ -304,6 +315,17 @@ Copyright (c) 2016-2019 Rigetti Computing.~2%")
             kind
             *available-allocation-kinds*))))
 
+(defun log-level-string-to-symbol (log-level)
+  (alexandria:eswitch (log-level :test #'string=)
```

Comment:
How do you feel about `assoc`ing over `cl-syslog::*priorities*` instead of replicating this list?

Reply:
I feel ok about it. If ever syslog's priorities should change, then this method would account for that automatically. However unlikely that is...